django/db/models/fields/json.py

```diff
@@ -310,6 +310,13 @@ def process_rhs(self, compiler, connection):
             rhs %= tuple(func)
         return rhs, rhs_params
 
+    def as_oracle(self, compiler, connection):
+        lhs, lhs_params = self.process_lhs(compiler, connection)
+        rhs, rhs_params = self.process_rhs(compiler, connection)
+        params = (*lhs_params, *rhs_params)
+        sql = "JSON_EQUAL(%s, %s)" % (lhs, rhs)
+        return sql, params
```

Comment:
Temporary variables are unnecessary:
```suggestion
        return f"JSON_EQUAL({lhs}, {rhs})", (*lhs_params, *rhs_params)
```

Reply:
done